Simple-Multiparty/build.gradle

```diff
@@ -15,9 +15,8 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
-        maven {
-            url  "http://tokbox.bintray.com/maven"
-        }
+        maven { url 'https://maven.google.com' }
```

Comment:
Agree. "The support libraries are now available through Google's Maven repository. You do not need to download the support repository from the SDK Manager."